src/pages/verify/guides/custom-styling.mdx

```diff
@@ -16,6 +16,15 @@ These may be styled and enhanced to you liking using one of the following approa
 1. [Adding your own stylesheet](#custom-css): if your goal is to apply basic styling, including a custom CSS file is all you need. 
 2. [Adding custom HTML and CSS](#custom-html): can be used for creating more complex UIs (e.g. a layout with extra information around the Criipto login box).
 
+<Highlight icon="info">
+
+This feature is available out of the box for [Criipto Verify](/verify) and [Criipto Signatures](/signatures) applications. 
```

Comment:
Doesn't fully work for Signatures, only the authentication part of signatures.

Reply:
Right, I'll omit this part.